packages/paste-website/src/pages/components/pagination/index.mdx

```diff
@@ -0,0 +1,560 @@
+
+<LivePreview scope={{Pagination, PaginationItems, PaginationArrow, PaginationLabel, Text}} language="jsx">
+  {`<>
+  <Pagination label="custom label pagination navigation">
+    <PaginationItems>
+      <PaginationArrow label="Go to previous month" variant="back" visibleLabel="Previous" />
+      <PaginationLabel>February 2021</PaginationLabel>
+      <PaginationArrow label="Go to next month" variant="forward" visibleLabel="Next" />
+    </PaginationItems>
+  </Pagination>
+  <Text as="span" display="block" fontStyle="italic" marginTop="space70">Will the "Next" button take you to January or March?</Text>
+</>`}
+</LivePreview>
+
+To give customers further clarity on where they are in a data set, you can add custom text to:
+
+- The current page label
+- The left and right direction controls
+
+<LivePreview scope={{Pagination, PaginationItems, PaginationArrow, PaginationLabel}} language="jsx">
+  {`<Pagination label="current page label pagination navigation">
+  <PaginationItems>
+    <PaginationArrow label="Go to newer templates" variant="back" visibleLabel="Newer templates" />
+    <PaginationLabel>51–100 of 500+ templates</PaginationLabel>
+    <PaginationArrow label="Go to older templates" variant="forward" visibleLabel="Older templates" />
+  </PaginationItems>
+</Pagination>`}
+</LivePreview>
+
+### Pagination with total page count
+
+Use the Pagination variant with total page count in cases when knowing the exact size of a dataset is crucial to a customer’s understanding of the content (e.g., knowing how many phone numbers aren’t compliant) and when you’re able to fetch the data in a performant way. This variant also allows customers to skip ahead pages in a dataset.
+
+<Callout>
+  <CalloutTitle>Considering the tradeoffs for showing the total count of a dataset</CalloutTitle>
+  <CalloutText>
+    By default,{' '}
+    <Anchor href="https://www.twilio.com/blog/replacing-absolute-paging-and-related-properties">
+      Twilio APIs might not support showing current page number or the total count of a dataset
+    </Anchor>
+    .
+  </CalloutText>
+  <CalloutText>
+    However, anecdotal evidence from product designers suggests that knowing the size of a dataset is important to
+    Twilio customers in certain cases. Showing the total count of a dataset makes sense especially when the customer has
+    more active control over total count, such as with total SIMs, but not for datasets that rapidly increase, like
+    event logs. If the exact total count is unknown, consider indicating size in another way that gives customers an
+    approximate idea of the size of a dataset, like showing "1–10 of 100+ results".
+  </CalloutText>
+</Callout>
+
+<LivePreview
+  scope={{
+    Pagination,
+    PaginationItems,
+    PaginationArrow,
+    PaginationNumbers,
+    PaginationNumber,
+    Table,
+    THead,
+    TBody,
+    Th,
+    Tr,
+    Td,
+    Box,
+  }}
+  noInline
+>
+  {numberExample}
+</LivePreview>
```

Comment:
In this example, can you show what the interaction would look like when there are ellipses? The ellipsis component's shown in the Usage Guide and Anatomy sections but mentioned nowhere else.

I'm not really sure how to use it (and if we'd need to add more guidelines around it).

Reply:
Added a section to the composition notes section with guidance and examples.